packages/react-components/react-virtualizer/library/src/components/Virtualizer/useVirtualizer.ts

```diff
@@ -438,14 +504,22 @@ export function useVirtualizer_unstable(props: VirtualizerProps): VirtualizerSta
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  /*
+   * forceUpdate:
+   * We only want to trigger this when scrollLoading is enabled and set to false,
+   * it will force re-render all children elements
+   */
+  const forceUpdate = useReducer(() => ({}), {})[1];
```

Comment:
Or was it this that we weren't sure if we needed?

Reply:
This one is required for when we need to trigger a re-render of children without state update in container, its only used for the isScrolling prop when activated.